resin/resin/imgui/sdf_tree.cpp

```diff
@@ -166,21 +169,27 @@ void SDFTreeComponentVisitor::visit_group(::resin::GroupNode& node) {
         if (ImGui::MenuItem("OBJ")) {
           ::resin::FileDialog::instance().save_file(
               [curr_id, &sdf_tree, resolution](const std::filesystem::path& path) {
-                ::resin::MeshExporter exporter(resolution);
+                ::resin::ShaderResourceManager resource_manager = ::resin::ResourceManagers::shader_manager();
```

Comment:
Avoid copying the resource manager:
```suggestion
auto& resource_manager = ::resin::ResourceManagers::shader_manager();
```

Reply:
done